src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/blob.py

```diff
@@ -124,3 +124,143 @@ def _copy_single_blob(source_blob):
 
     return [BlobCopyResult(b.name, _copy_single_blob(b).id) for b in source_blobs]
 
+
+#pylint: disable=unused-argument
+def storage_blob_download_batch(client, source, destination, source_container_name, pattern=None,
```

Comment:
Can pattern not simply be parsed from the source URI?  ` --source C://mypath/mydir/*.vhd`?